iotlabcli/integration/test_cli_tools_help.py

```diff
@@ -0,0 +1,36 @@
+import pytest
```

Comment:
The test you mention in tox.ini does not cover the use-cases that the pytest test does.  I have seen this bash for loop and decided to provide a systematic, "as a user would do" test case, of all cli tools menus/sub-menus, thus the pytest.